src/components/panels/InvasionsPanel.vue

```diff
@@ -53,6 +37,13 @@
       headertext() {
         return 'Invasions';
       },
+      maxInvasions() {
+        return 5;
+      },
+      initialStatus() {
+        var state = this.$store.getters.componentState['invasions'];
```

Comment:
this can just be `this.$store.getters.componentState.invasions` since it doesn't have any special characters